src/openloco/ui/dropdown.cpp

```diff
@@ -60,9 +70,24 @@ namespace openloco::ui::dropdown
         add(index, title, { l });
     }
 
+    int16_t get_highlighted_item()
+    {
+        return _dropdownHighlightedIndex;
+    }
+
+    void set_disabled_item(int16_t index)
+    {
+        _dropdownDisabledItems = _dropdownDisabledItems | (1 << index);
+    }
+
+    void set_highlighted_item(int16_t index)
+    {
+        _dropdownHighlightedIndex = index;
+    }
+
     void set_selection(int16_t index)
     {
-        _dropdownSelection = (1 << index);
+        _dropdownSelection = _dropdownSelection | (1 << index);
```

Comment:
Same as previous

Reply:
You mean to say rename this function to `set_item_selected`?